sharding/nodesCoordinator/indexHashedNodesCoordinator_test.go

```diff
@@ -1797,14 +1872,22 @@ func TestIndexHashedNodesCoordinator_ConsensusGroupSize(t *testing.T) {
 	t.Parallel()
 
 	arguments := createArguments()
+	arguments.ChainParametersHandler = &shardingmock.ChainParametersHandlerStub{
+		CurrentChainParametersCalled: func() config.ChainParametersByEpochConfig {
+			return config.ChainParametersByEpochConfig{
+				ShardConsensusGroupSize:     1,
```

Comment:
can you use a variable to initialize here the group size, and then use the same variables in the requires below as expected values

Reply:
done